<div>                      </div><div>One of the key challenges in systems biology is to characterize the energetic state of the cell. However, unlike transcriptomics and proteomics data sets, almost all metabolomics data sets are sparse. Part of the reason for this sparsity is that the chemical properties of small molecules vary widely, so no single instrument can measure every type of metabolite.&nbsp; Sparsity is also due to the fact that biologically relevant metabolite &nbsp;concentrations may span several orders of magnitude, which is greater than what typical instrument sensitivities can handle. Therefore, new computational and theoretical methods are needed that can interpolate unmeasured or unmeasurable metabolite levels within a metabolic network using the least biased assumptions that are consistent with our current knowledge.</div><div>Constraint-based methods such as flux balance analysis (FBA) have been successfully used to predict the steady-state metabolic fluxes at genome-scale, but net flux predictions by themselves provide no information about the metabolite concentrations. Conversely, kinetic models can be used to predict steady-state metabolite concentrations, but cannot generally be applied at the genome scale due to the challenges involved in measuring rate constants in a high-throughput manner. Thermodynamic-based methods have been used to constrain the reaction direction of metabolic fluxes or to remove thermodynamically infeasible cycles. More recently, an optimization method has been developed that can predict metabolite concentrations by hypothesizing that a metabolic tug of war <cite class="ltx_cite raw v1">\cite{Tepper2013}</cite> exists between osmotic pressure to keep metabolite concentrations low, and the need to maintain thermodynamic non-equlibrium in order utilize enzymes efficiently. </div><div>An alternative approach for modeling metabolism is to predict energetic states where the  ratio of the forward and backward rates is simulated using non-equilibrium thermodynamics. In this approach, we showed that the likelihood ratio of the forward and backward reaction rates are proportional to the thermodynamic driving force<cite class="ltx_cite raw v1">\cite{Cannon2014}</cite>.&nbsp;In a recent study of the coupled reaction theorem, we demonstrated that both thermodynamic and kinetic optimal states are obtained at the maximum rate of entropy production&nbsp;<cite class="ltx_cite raw v1">\cite{Cannon2017}</cite>..&nbsp;</div><div>Building on this body of work, we present Maximum ENtropy production rate Of the Stoichiometric matrix (MENTOS): a statistical thermodynamic optimization method for predicting the metabolite concentrations and fluxes that extract energy from the environment as quickly and efficiently as possible, given the constraints of the network.&nbsp; MENTOS was developed to address two major goals. First, the ability to predict reasonable values for steady state metabolite concentrations can be used as a foundation for developing thermokinetic models that can predict non-steady-state dynamics in the absence of rate constants. Second, MENTOS provides a quantitative measure by which we can test the hypothesis that natural systems have evolved to move to the most probable state through the fastest path by minimizing energetic costs. &nbsp;In the theory section below, we develop the statistical thermodynamic concepts necessary to formalize this hypothesis, and demonstrate the application of this hypothesis to a simple toy model. We will then apply the hypothesis to the central carbon metabolism of E. coli and compare the metabolite predictions with actual measurements.</div><div>&nbsp;<b>Theory/Methods:&nbsp;</b>In this section, we will formalize the hypothesis described ab provides the &nbsp;basic concepts of statistical thermodynamics that are required for the formulation of the MENTOS approach.&nbsp;</div><div>  </div><div>&nbsp;</div><div>&nbsp;</div><div><b>2.1&nbsp; Relationship between reaction rates and concentrations: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </b></div><div>  </div><div>Consider a simple bimolecular reversible reaction in which&nbsp;<span class="math ltx_Math v1">\(\gamma_A\)</span> moles of<span class="math ltx_Math v1">\(A\)</span> and  moles of <span class="math ltx_Math v1">\(B\)</span> are transformed into <span class="math ltx_Math v1">\(\gamma_C\)</span> moles of C and <span class="math ltx_Math v1">\(\gamma_D\)</span> moles of <span class="math ltx_Math v1">\(D\)</span>. </div><div>This reaction proceeds at forward reaction rate r+ and reverse reaction rate r- :</div><div>  </div>